OneginiExampleAppSwift/Interactors/MobileAuthInteractor.swift

```diff
@@ -139,7 +130,8 @@ class MobileAuthInteractor: NSObject, MobileAuthInteractorProtocol {
         if mobileAuthEntity.cancelled {
             biometricChallenge.sender.cancel(biometricChallenge)
         } else {
-            biometricChallenge.sender.respondWithDefaultPrompt(for: biometricChallenge)
+            //TODO: what to do here?
+//            biometricChallenge.sender.respondWithDefaultPrompt(for: biometricChallenge)
```

Comment:
We need to send an example prompt. Im not sure what it should be. Previously we had something like `Please provide your fingerprint` but now it will not be good for Face ID. Also its not conforming to apple recommendations since we should describe the reason `Providing this prompt lets the user know why you want them to authenticate.` (https://developer.apple.com/documentation/localauthentication/accessing_keychain_items_with_face_id_or_touch_id). 
So I guess the prompt should go more like `User authentication`